ext/afform/gui/xml/Menu/afform_gui.xml

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Why did this just get removed?

Reply:
Per the code comment I added, the path `civicrm/admin/afform` doesn't need to be registered in Civi's menu as it's registered as an afform and handled via afform's magical routing.
We made that switch a while ago, this is just cruft.